lib/serverBuilder.js

```diff
@@ -0,0 +1,100 @@
+/*!
```

Comment:
Instead of using `*Builder` as our pattern, what do you think about `Client`? To me, **client** is a bit more descriptive.

```
lib/browserClient.js
lib/serverClient.js
```

Reply:
I think `serverClient` sounds a bit odd, doesn't it?

Reply:
If so, how about we just use `server` and `browser`? They're internal files that _should_ be descriptive enough. What do you think?

Reply:
I like server/browser - while I'll normally argue for added context in names, it isn't worth it if the context doesn't increase clarity.